### DISCUSSION TITLE
What in the activity table?

### DISCUSSION THREAD

**Discussion by Merca60** [community/community#123282] [New to GitHub] [answered]
Two days ago I created a comment, why it isn't in my activity table?

**Comment by davevad93** [comment]
Hi @Merca60 ,

The GitHub docs states the following regarding [what counts as a contribution](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/viewing-contributions-on-your-profile#what-counts-as-a-contribution):

`On your profile page, certain actions count as contributions:`

`- Committing to a repository's default branch or gh-pages branch`
`- Creating a branch`
`- Opening an issue`
`- Opening a discussion`
`- Answering a discussion`
`- Proposing a pull request`
`- Submitting a pull request review`

Making a comment in a `pull request` or in a `discussion` doesn't count as a contribution.

**Reply by Merca60** [reply]
OK, thanks.

**Reply by davevad93** [reply, marked as answer]
You're welcome @Merca60 , If you think I answered your question properly, please, mark it as `answered`.

**Comment by Merca60** [comment]
Fatto.. 

Sto imparando...

    Il venerdì 10 maggio 2024 alle ore 22:31:27 CEST, Davide Presti ***@***.***> ha scritto:  
 
 


You're welcome @Merca60 , If you think I answered your question properly, please, mark it as answered.
